astropy/io/fits/hdu/image.py

```diff
@@ -1087,6 +1090,9 @@ def __init__(self, data=None, header=None, name=None,
             operations on the data.  Pseudo-unsigned integers are automatically
             rescaled unless scale_back is explicitly set to `False`.
             (default: None)
+
+        ver : int > 0, optional
+            The name of the HDU, will be the value of the keyword ``EXTVER``.
```

Comment:
Maybe add a note that, if not given, it will be set to 1 even in the case of duplicate `EXTNAME` in a HDU list?